tests/fun3d_tests/fun3d_aero_remesh/test_fun3d_remesh.py

```diff
@@ -68,7 +67,7 @@ def test_optimizer_file(self):
         ).register_to(wing)
         wing.register_to(model)
         test_scenario = (
-            Scenario.steady("turbulent", steps=10)
+            Scenario.steady("euler", steps=10)
```

Comment:
Are you sure you want to switch to Euler?

Reply:
See above comment on mismatching folders.